doc/man3/ASN1_INTEGER_new.pod

```diff
@@ -0,0 +1,44 @@
+=pod
+
+=head1 NAME
+
+ASN1_INTEGER_new, ASN1_INTEGER_free, ASN1_INTEGER_it -
+ASN1_INTEGER allocation functions
+
+=head1 SYNOPSIS
+
+=for openssl generic
+
+ #include <openssl/asn1.h>
+
+ extern const ASN1_INTEGER_it;
+ ASN1_INTEGER *ASN1_INTEGER_new(void);
+ void ASN1_INTEGER_free(ASN1_INTEGER *a);
+
+=head1 DESCRIPTION
+
+ASN1_INTEGER_new() returns an allocated B<ASN1_INTEGER> structure.
```

Comment:
But what *is* an ASN1_INTEGER structure? It should say something about what it represents.

Reply:
fixed